apis/swagger.yml

```diff
@@ -1964,6 +1964,10 @@ definitions:
       QuotaID:
         type: "string"
         description: "set disk quota by specified quota id, if id < 0, it means pouchd alloc a unique quota id"
+      NetPriority:
+        description: "net priority."
```

Comment:
Could you add more description of this field? More detailed content makes more people know more about the great things you are doing.

Reply:
No, it will be `TODO` things.